windows/LightGBM.vcxproj

```diff
@@ -34,7 +34,7 @@
     <SccLocalPath>SAK</SccLocalPath>
     <SccProvider>SAK</SccProvider>
     <ProjectName>LightGBM</ProjectName>
-    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
```

Comment:
This is the default version installed together with VS2017 from this link https://aka.ms/vs/15/release/vs_community.exe as of time of writing.